py/create_and_estimate_audience.py

```diff
@@ -0,0 +1,134 @@
+#
+# Create a reach estimate for an audience and save the resulting audience. 
+# This example creates and audience of the form "(A OR B) AND C" where A,B, and C are chosen dynamically as follows:
+#
+# A = first node from the Lotame standard hierarchy, using only first party data
+# B = second node from the Lotame standard hierarchy, using first and third party data
+# C = first node from the first branded hierarchy returned by the API.
+#
+# The audience defaults to "Enrich", which means overlap with the clients inventory. Use the argument -extend to build
+# the audience on "Extend".
+#
+# Note: the ids used in this example may not be accessible to all user accounts.
+# 
+import datetime, time, sys, json, argparse, lotame_utils
+
+# force everything to play nice for shell redirects, etc...
+reload(sys)
+sys.setdefaultencoding("utf-8")
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('-user', help='API username', required=True)
+parser.add_argument('-pw', help='API password', required=True)
+parser.add_argument('-clientid', help='ID of client for which to scope all API calls', required=True)
+parser.add_argument('-extend', help='Build the audiences on extend instead of enrich', action='store_true')
+args = parser.parse_args()
+
+# Get hierarchies
+hierarchyList = lotame_utils.getRequest(args.user, args.pw, "https://api.lotame.com/2/hierarchies?view=LAB&client_id=" + args.clientid).json()
+
+# Find the nodes we want to use. The standard hierarchies are identified by default=true, while
+# a branded hierarchy has a hierarchyType of "BRANDED", and the ownerClientId represents the client from which data is purchased
+nodeA = None
+nodeB = None
+nodeC = None
+for hierarchy in hierarchyList["hierarchies"]:
+	if nodeA == None and hierarchy["default"] == True:
+		# this is the standard hierarchy, grab the first two nodes for our A or B expression that we'll build later
+		hierarchyMeta = lotame_utils.getRequest(args.user, args.pw, "https://api.lotame.com/2/hierarchies/" + hierarchy["id"] + "?depth=2&universe_id=1&client_id=" + args.clientid).json()
+		nodes = hierarchyMeta["nodes"]
+		if nodes != None:
+			nodeA = {"clientId":hierarchyMeta["ownerClientId"],"node":nodes[0]}
+			nodeB = {"clientId":hierarchyMeta["ownerClientId"],"node":nodes[1]}
```

Comment:
yeah, i'm ok with that. We note in the readme for these examples that it's missing almost all error handling